main.js

```diff
@@ -1,8 +1,9 @@
 var table = document.querySelector("table");
-var gradeTable = new GradeTable(table);
+var gradeTable = new GradeTable(table, noGradesRecorded);
 var header = document.querySelector("header");
 var pageHeader = new PageHeader(header);
 var form = document.querySelector("form");
 var gradeForm = new GradeForm(form);
+var noGradesRecorded = document.querySelector("p");
```

Comment:
Since `<p>`s show up pretty frequently in HTML docs, please use a more specific selector (_hint: it already has an id_).